Resources/translations/SonataFormatterBundle.cs.xliff

```diff
@@ -6,6 +6,10 @@
                 <source>please_select_format_method</source>
                 <target>vyberte možnost formátování textu</target>
             </trans-unit>
+            <trans-unit id="sonata_formatter_block_formatter">
+                <source>sonata.formatter.block.formatter</source>
+                <target>Rich Text Area</target>
```

Comment:
if you don't have the correct translation, please use the ``source`` as ``target`` and NOT the english one.

thank you